### PR TITLE
React react1 week3

### PR DESCRIPTION
**Pull request by oleksandra-lavryk**

### CODE REVIEW

File: react/react1/week3/todo-list/src/components/ToDoList.jsx [hunk 1]
```diff
@@ -95,3 +105,10 @@ export default function ToDoList() {
     </>
   );
 }
+
+TaskItem.propTypes = {
+  checked: PropTypes.bool,
+  description: PropTypes.string,
+  deadline: PropTypes.string,
+  handleInputChange: PropTypes.func,
+};
```
Comment (MaherHussain) [inline react/react1/week3/todo-list/src/components/ToDoList.jsx:138]: grate to add propTypes :-)

Comment (MaherHussain) [inline react/react1/week3/todo-list/src/components/ToDoList.jsx:138]: > great adding propTypes :-)

**Review by MaherHussain — APPROVED** [review]

**Comment by MaherHussain** [comment]
great job . thanks Oleksandra :-) feel free to merge your BR

**Review by MaherHussain — APPROVED** [review]

**Comment by oleksandra-lavryk** [comment]
> great job . thanks Oleksandra :-) feel free to merge your BR
Thanks a lot :)

**Pull request merged**